solaris/solaris11/generate_wazuh_packages.sh

```diff
@@ -99,12 +101,20 @@ compile() {
     arch="$(uname -p)"
     # Build the binaries
     if [ "$arch" = "sparc" ]; then
-        gmake -j $THREADS TARGET=agent PREFIX=${install_path} USE_SELINUX=no USE_BIG_ENDIAN=yes
+        if [ "${disa_share}" == "true" ]; then
```

Comment:
This `if/else` block can be simplified if you set the `disa_share` variable to `yes` or `no` and then you just need to run this line `gmake -j $THREADS TARGET=agent DISABLE_SHARED=${disa_share} PREFIX=${install_path} USE_SELINUX=no USE_BIG_ENDIAN=yes`.

In addition, please change the name of `disa_share` to something more clear.

Reply:
Change